uxarray/grid.py

```diff
@@ -528,6 +540,34 @@ def _build_edge_node_connectivity(self):
         setattr(self, "Mesh2_edge_nodes", self.ds['Mesh2_edge_nodes'])
         setattr(self, "nMesh2_edge", edge_nodes_unique.shape[0])
 
+    def _build_face_edges_connectivity(self):
+        """Constructs the UGRID connectivity variable (``Mesh2_face_edges``)
+        and stores it within the internal (``Grid.ds``) and through the
+        attribute (``Grid.Mesh2_face_edges``)."""
+        if "Mesh2_edge_nodes" not in self.ds.keys(
+        ) or (self.ds['Mesh2_edge_nodes'].attrs.get("inverse_indices") is None):
```

Comment:
We can tidy up this check by doing the following:
```suggestion
        if ("Mesh2_edge_nodes" not in self.ds or  
            "inverse_indices" not in self.ds['Mesh2_edge_nodes'].attrs):
```